--- gmime-2.4.33/configure.in.0001~	2012-11-03 17:23:04.000000000 +0100
+++ gmime-2.4.33/configure.in	2013-01-15 22:08:03.845415960 +0100
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(configure.in)
-AM_CONFIG_HEADER(config.h)
+AC_INIT(gmime, 2.4.33)
+AC_CONFIG_HEADERS([config.h])
 
 # In the following, there are a the following variants
 # of GLib cflags and libs variables
@@ -28,7 +28,7 @@ AC_SUBST(GMIME_API_VERSION)
 
 VERSION=$GMIME_VERSION
 
-AM_INIT_AUTOMAKE(gmime, $VERSION)
+AM_INIT_AUTOMAKE
 
 AC_CANONICAL_HOST
 
@@ -66,7 +66,6 @@ fi
 dnl Checks for programs.
 AC_ISC_POSIX
 AC_PROG_CC
-AM_PROG_CC_STDC
 AC_C_INLINE
 dnl AC_HEADER_STDC
 AC_STDC_HEADERS
